src/lib/signerVerifier.js

```diff
@@ -25,12 +25,28 @@ class Ed25519Verifier {
   }
 
   verify(vc) {
+    // eslint-disable-next-line no-bitwise
+    return this.verifyHex(vc) || this.verifyUtf(vc);
+  }
+
```

Comment:
On a side note: is there any information available to let us know if the encoding is in hex/UTF8?

Not that I can see... the signatures are the same length, and no real identitying information available.

This will no longer exist once we move this out and it will be up to the implementor